.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Thank you for using RabbitMQ and for taking the time to report an issue.
+
+*Important:* please first read the [`CONTRIBUTING`](../CONTRIBUTING.md) document in the root of this repository to be sure your issue shouldn't be discussed on the [mailing list][rmq-users] first.
+
+In order for the RabbitMQ team to investigate your issue, the following *must* be provided:
+  
+* Erlang version
+* RabbitMQ version
+* RabbitMQ plugin version(s)
+* Client library version (for all libraries used)
+* Operating system, version, and patchlevel
+* RabbitMQ, server and client application logs
+* A *complete*, working code sample, terminal transcript or set of instructions that can be used to reproduce the issue. We can't stress enough how much this helps!
```

Comment:
I'd say "a reasonably complete runnable code example or a detailed step-by-step description of user/operator actions".